solidity/test/integration/ConnextVestingWallet.t.sol

```diff
@@ -7,16 +7,21 @@ import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
 import {ConnextVestingWallet} from 'contracts/ConnextVestingWallet.sol';
 import {IntegrationBase} from 'test/integration/IntegrationBase.sol';
 
+/**
+ * TODO: Make this tests more generic and clear
+ */
 contract IntegrationConnextVestingWallet is IntegrationBase {
```

Comment:
blz do a total amount of `13 ether` and show the 1/13s behaviour with `1 ether` additions each month